src/main/java/org/apache/commons/lang3/StringUtils.java

```diff
@@ -9585,6 +9587,171 @@ public static String wrapIfMissing(final String str, final String wrapWith) {
+     */
+    public static String toTitleCase(String str) {
+        if (str == null) {
+            return null;
+        }
+        String[] words = str.split("\\s");
+        StringBuilder titleCase = new StringBuilder();
+        for (String word : words) {
+            if (word.length() > 0) {
+                titleCase.append(Character.toUpperCase(word.charAt(0)))
+                         .append(word.substring(1).toLowerCase())
+                         .append(" ");
+            }
+        }
+        return titleCase.toString().trim();
+    }
+
+    /**
+     * Finds the longest common prefix of two Strings.
+     *
+     * @param str1 the first String
+     * @param str2 the second String
+     * @return the longest common prefix
+     */
+    public static String longestCommonPrefix(String str1, String str2) {
+        if (str1 == null || str2 == null) {
+            return null;
+        }
+        int minLength = Math.min(str1.length(), str2.length());
+        for (int i = 0; i < minLength; i++) {
+            if (str1.charAt(i) != str2.charAt(i)) {
+                return str1.substring(0, i);
+            }
+        }
+        return str1.substring(0, minLength);
+    }
+
+    /**
+     * Repeats a String n times.
+     *
+     * @param str the String to repeat
+     * @param n the number of times to repeat the String
+     * @return the repeated String
+     */
+    public static String repeat(String str, int n) {
+        if (str == null) {
+            return null;
+        }
+        StringBuilder repeatedStr = new StringBuilder();
+        for (int i = 0; i < n; i++) {
+            repeatedStr.append(str);
+        }
+        return repeatedStr.toString();
+    }
+
+    /**
+     * Checks if a String contains only digits.
+     *
+     * @param str the String to check
+     * @return true if the String contains only digits, false otherwise
+     */
+    public static boolean isNumeric(String str) {
+        if (str == null) {
+            return false;
+        }
+        return str.matches("\\d+");
+    }
+
+    /**
+     * Capitalizes the first letter of each word in a String.
+     *
+     * @param str the String to capitalize
+     * @return the capitalized String
+     */
+    public static String capitalizeWords(String str) {
+        if (str == null) {
+            return null;
+        }
+        String[] words = str.split("\\s");
+        StringBuilder capitalized = new StringBuilder();
+        for (String word : words) {
+            if (word.length() > 0) {
+                capitalized.append(Character.toUpperCase(word.charAt(0)))
+                           .append(word.substring(1))
+                           .append(" ");
+            }
+        }
+        return capitalized.toString().trim();
+    }
+
+    /**
+     * Converts a String to snake_case.
+     *
+     * @param str the String to convert
+     * @return the snake_case String
+     */
+    public static String toSnakeCase(String str) {
```

Comment:
Hello @GERONlMO 
New case functions and any new text related (as opposed to low-level String operations) code is only being considered for Commons Text, see PRs in that repository.